modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -242,12 +255,35 @@ inline void preprocess(const cv::Mat& src,
             dst = pp;
         }
 
+        if (with_pad) {
+            const auto div = ti.pad->divisibility;
+            const int padded_h = std::ceil(new_h / div) * div;
+            const int padded_w = std::ceil(new_w / div) * div;
+            int type = -1;
+            // is_hwc - Mat has C*H, W dimensions
```

Comment:
This comment should be vice versa, right? if Mat is_not_hwc, i. e. CHW => dimensions [C*H, W, 1]

Reply:
Removed after "padding" discussion.